scripts/install-nix-from-closure.sh

```diff
@@ -108,8 +109,13 @@ if [ "$(uname -s)" = "Darwin" ]; then
         "$self/create-darwin-volume.sh"
     fi
 
-    info=$(diskutil info -plist / | xpath "/plist/dict/key[text()='Writable']/following-sibling::true[1]" 2> /dev/null)
-    if ! [ -e $dest ] && [ -n "$info" ] && [ "$macos_major" -gt 14 ]; then
+    writable="$(diskutil info -plist / | xmllint --xpath "/plist/dict/key[text()='Writable']/following-sibling::*[1]/text()" -)"
+    # TODO: I wonder a bit if we need the version test, here. This may seem odd since I
```

Comment:
I think I suggested removing it, just checking if it's writable should be enough.

Reply:
Tentatively addressed in 30bd194, but leaving this open as a reminder to remove the comment once I've confirmed it works.